content/blog/entries/cc-chooser-week2/contents.lr

```diff
@@ -0,0 +1,26 @@
+ title: GSoC cc-chooser Week 2
+---
+ categories:
+ gsoc
+ gsoc-2019
+---
+ author: akmadian
+---
+ pub_date: 2019-06-14
+---
+In case you don't know, my GSoC project involves revamping [the CC license chooser](https://creativecommons.org/choose/).
```

Comment:
You're missing the `body` key here so your post is showing up as empty.
![Screenshot_2019-06-14 GSoC cc-chooser Week 2 — Creative Commons on GitHub](https://user-images.githubusercontent.com/287034/59534905-8aca7d00-8ebd-11e9-8afa-4eed64adfd45.png)

Reply:
Whoops :/ Fixed!